spec/unit/navigation/navigation.spec.js

```diff
@@ -114,6 +133,19 @@ describe("navigation toggle", () => {
     assert.strictEqual(accordionButton.getAttribute("aria-expanded"), "false");
   });
 
+  it("collapses dropdowns when the Escape key is hit", () => {
```

Comment:
Unit test is now more specific. When you open a dropdown it should close on <kbd>Esc</kbd>.